### DISCUSSION TITLE
Add ability to add custom networks, change RPCs

### DISCUSSION THREAD

**Discussion by StackOverflowExcept1on** [phantom/docs#261] [General]
Currently in Settings -> Developer Settings -> Testnets -> Ethereum you cannot select testnets other than Sepolia and Polygon Amoy. It would be nice to add local testnet, similar to Solana. Also, the wallet is currently supports only 2 EVM networks. It would be nice to add L2 solutions to the wallet or add the ability to add any EVM-compatible network.

**Comment by adamdelphantom** [comment]
Hey @StackOverflowExcept1on!

Thank you for writing in! Please weigh in on your support for the custom rpc feature [here](https://phantom.canny.io/feature-requests/p/add-support-for-custom-rpc-endpoints).

And if you have any other chains you would like to see please add those requests or vote for them. Our product team is constantly considering user support for various features..
